src/features/network-drawer/networks-drawer.tsx

```diff
@@ -13,6 +13,7 @@ import { Caption, Title } from '@components/typography';
 import { getUrlHostname } from '@common/utils';
 import { useAtomValue, useUpdateAtom } from 'jotai/utils';
 import { IconCloudOff } from '@tabler/icons';
+import { SettingsSelectors } from '../../../tests/integration/settings.selectors';
```

Comment:
We should prob make an alias for tests at some point.

Reply:
Yeah, here we'd ideally also have a depcruiser rule that **only** allows `*.selector` files.

Reply:
> We should prob make an alias for tests at some point.

I tried but got stuck on how best to make jest use it, but yeah it would be great!